src/components/Editor/index.js

```diff
@@ -568,14 +570,22 @@ const Editor = React.createClass({
       }));
   },
 
-  editorHeight() {
-    const { selectedSource, horizontal } = this.props;
+  getInlineEditorStyles() {
+    const { selectedSource, horizontal, searchOn } = this.props;
 
+    // The footer is off if there is no 'prettify' button (which happens for
+    // source-mapped or already prettified files)
     if (!shouldShowFooter(selectedSource, horizontal)) {
-      return "100%";
+      if (searchOn) {
+        return { height: "calc(100% - 54px)" };
```

Comment:
So... It's not great that we have a *magic* number here because it will definitely change.

I don't love suggesting this, but it might be helpful to calculate the heights of the footer and search bar...

Here's where we get calculated DOM dimensions elsewhere: 

https://github.com/devtools-html/debugger.html/search?utf8=%E2%9C%93&q=getBoundingClientRect&type=